### ISSUE TITLE
src and jar dates discrepancy

### ISSUE THREAD

**Issue by michaeldav** [edmund-wagner/junrar#10]
Hi
I am curious the sources dates to 29/11/2011 but the latest 0.5 jar to 30/01/11. The jar appears to be is much earlier?
Is this code match to the jar?

Can you please clarify this?

Thanks
Michael

**Comment by edmund-wagner** [comment]
yes the jar is quite old. i'm about to put junrar into repo1 but no private inet at the moment....
